lib/pentagon/Makefile

```diff
@@ -72,13 +91,15 @@ destroy_vpn:
 
 destroy_vpn_bucket:
 	source config/local/env-vars.sh; \
-	aws s3api list-object-versions --bucket  $(VPN_BUCKET) | jq -r '.Versions[] | .VersionId + " " + .Key' > .tmp ; \
+	aws s3 ls s3://$(VPN_BUCKET) || exit 0 && \
```

Comment:
What's the reasoning for `|| exit 0 &&`?

Reply:
If it fails to list the bucket, it assumes the bucket does not exist and exits this target. If the bucket does not exist, then the target fails and stops the run.